src/views/Provisioners/ViewWorkers/index.jsx

```diff
@@ -143,25 +149,24 @@ export default class ViewWorkers extends Component {
                   workerType={params.workerType}
                   provisionerId={params.provisionerId}
                 />
-                <SpeedDial
-                  title={
-                    workerType.actions.length ? null : 'No actions available'
-                  }>
-                  {workerType.actions.map(action => (
-                    <SpeedDialAction
-                      key={action.title}
-                      ButtonProps={{ color: 'secondary' }}
-                      icon={<HammerIcon />}
-                      tooltipTitle={
-                        <div>
-                          <div>{action.title}</div>
-                          <div>{action.description}</div>
-                        </div>
-                      }
-                      onClick={() => this.handleActionClick(action)}
-                    />
-                  ))}
-                </SpeedDial>
+                {workerType.actions.length ? (
+                  <SpeedDial>
+                    {workerType.actions.map(action => (
+                      <SpeedDialAction
+                        key={action.title}
+                        ButtonProps={{ color: 'secondary' }}
+                        icon={<HammerIcon />}
+                        tooltipTitle={
+                          <div>
+                            <div>{action.title}</div>
+                            <div>{action.description}</div>
```

Comment:
Remove description. The description will be in the dialog that gets opened, and putting it here will be potentially too large and redundant to show every time.